deepmd_utils/utils/data_system.py

```diff
@@ -233,12 +234,18 @@ def _load_test(self, ntests=-1):
     @lru_cache(maxsize=None)
```

Comment:
It's cached, so I am wondering whether the returned value can be changed.

Reply:
I think it works as expected. I have a simple test case:
```python
from typing import List
import numpy as np
from functools import lru_cache


class Tester:
    def __init__(self):
        self._default_mesh = None

    @property
    @lru_cache(maxsize=None)
    def default_mesh(self) -> List[np.ndarray]:
        """Mesh for each system."""
        if self._default_mesh is None:
            self._default_mesh = [np.ones(1), np.ones(1)]
        return self._default_mesh

    @default_mesh.setter
    def default_mesh(self, value: List[np.ndarray]):
        self._default_mesh = value


obj = Tester()
print(obj.default_mesh)
tmp_mesh = [np.zeros(1), np.zeros(1)]
obj.default_mesh += tmp_mesh
print(obj.default_mesh)
obj.default_mesh += tmp_mesh
print(obj.default_mesh)
```
which returns:
```text
[array([1.]), array([1.])]
[array([1.]), array([1.]), array([0.]), array([0.])]
[array([1.]), array([1.]), array([0.]), array([0.]), array([0.]), array([0.])]
```

Reply:
It seems that `+=` will not change the pointer to an object, so it's reasonable.

```py
>>> a=[]
>>> id(a)
140386313046464
>>> a+=[5]
>>> id(a)
140386313046464
```